rlclientlib/live_model_impl.cc

```diff
@@ -247,16 +253,25 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
-  int live_model_impl::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+  int live_model_impl::request_multi_slot_decision(const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, const std::vector<int>& baseline_actions, api_status* status)
   {
     const auto uuid = boost::uuids::to_string(boost::uuids::random_generator()());
-    return request_multi_slot_decision(uuid.c_str(), context_json, flags, resp, status);
+    return request_multi_slot_decision(uuid.c_str(), context_json, flags, resp, baseline_actions, status);
   }
-  
-  int live_model_impl::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, api_status* status)
+
+  int live_model_impl::request_multi_slot_decision(const char * event_id, const char * context_json, unsigned int flags, multi_slot_response_detailed& resp, const std::vector<int>& baseline_actions, api_status* status)
   {
     resp.clear();
 
+    if (_learning_mode == APPRENTICE)
+    {
+      if (baseline_actions.size() == 0)
+      {
+        return error_code::baseline_actions_not_defined;
+      }
+      //TODO: validity check for baseline actions
```

Comment:
Is there a task for this todo?

Reply:
We decided not to have a basline action validity check on the rl client side. I forgot to remove the comment.